tests/random_test.py

```diff
@@ -1875,6 +1876,18 @@ def test_make_array_from_single_device_arrays(self):
     result = jax.make_array_from_single_device_arrays(shape, sharding, arrays)
     self.assertArraysEqual(result, keys)
 
+  def test_multihost_utils(self):
```

Comment:
Note that there's no point in adding a test here.

The test should probably live in a different file internally (which I can't say here)

Reply:
Sounds good, thanks